### ISSUE TITLE
Many live updates are easy to miss

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#136]
During a user test today, we noticed an issue with the way AT handles aria-live updates.

The `aria-live` update is cut short if the user continues to interact with the keyboard while the updated text is read.  This poses a huge challenge for interactives because _most_ of our descriptions will be read either during or while a user is interacting with some component.  Many live descriptions will be easy to miss.

**Comment by terracoda** [comment]
@jessegreenberg I see this in my interviews as well. When a user really rubs on the Sweater actively, they don't actually hear the live updates. Some participants paused when hearing an update start, but a few of the really active ones didn't. They just kept going. I have been thinking that it would be nice to have an on-demand way to check charge levels and or position. 
Is it possible that some information can be provide with alerts, a special case live region. Are you already mixing these type special case live regions in with aria-live assertive? aria-live assertive and alerts have the same priority level, so I am not sure how they will work together.

**Comment by jessegreenberg** [comment]
Yes, we could try alerts.  We haven't tried alerts yet for live regions.  We should avoid the Window alert() method if possible, this would not integrate well with non-screen reader users, and the window event behaves like a dialog.  But we should try the alert role, perhaps this role will not cancel with keyboard interaction.

**Comment by jessegreenberg** [comment]
aira-live alerts are now fired on keyup in this sim.  This helps with this issue because the user is encouraged to listen for information after completion of an interaction.  This has been very helpful for alerts during keyboard interaction.  But since the alerts are not linked directly to changing properties, we currently do not hear anything when interacting with a mouse. A new issue should be created for this. Closing.
